extensions/ql-vscode/src/remote-queries/view/SortRepoFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { FilterIcon } from '@primer/octicons-react';
+import { ActionList, ActionMenu } from '@primer/react';
+
+export type Sort = 'name' | 'stars' | 'results';
+type SortBy = { name: string, sort: Sort }[];
+type Props = {
```

Comment:
Personal preference (feel free to ignore) but I've generally avoided having a `Props` type because it makes you jump around in the code - I just define it inline where the component that has the props is defined.

This helps if you have multiple components in the same file too - avoids the reader having to wonder for which component the props are for.